back/src/main/java/com/speakerspace/controller/EventController.java

```diff
@@ -0,0 +1,177 @@
+package com.speakerspace.controller;
+
+import com.speakerspace.dto.EventDTO;
+import com.speakerspace.exception.EntityNotFoundException;
+import com.speakerspace.exception.UnauthorizedException;
+import com.speakerspace.security.AuthenticationHelper;
+import com.speakerspace.service.EventService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import java.nio.file.AccessDeniedException;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/event")
+@RequiredArgsConstructor
+public class EventController {
+
+    private final EventService eventService;
+    private final AuthenticationHelper authHelper;
+
+    @PostMapping("/create")
```

Comment:
suggestion: mettre des annotations SpringDoc 

doc : https://springdoc.org/#demos
exemple : https://github.com/springdoc/springdoc-openapi-demos/blob/master/demo-spring-boot-3-webmvc/src/main/java/org/springdoc/demo/app2/api/PetApi.java

En l'état si t'ouvre le swagger tu dois avoir ta liste de endpoint mais t'as pas de doc : pas de détail sur les codes d'erreurs à minima.

Perso j'aurai ajouter au moins quelque chose dans ce genre là :

```suggestion
    @Operation(summary = "Create a new event")
	@ApiResponses(value = {
			@ApiResponse(responseCode = "200", description = "Successful operation", content = { @Content(schema = @Schema(implementation = EventDTO.class)) }),
			@ApiResponse(responseCode = "401", description = "Not authorized")
	})
	@PostMapping("/create")
```

Sachant que par exemple 401 si tu peux l'avoir sur chaque endpoint de la class, tu peux mettre la doc sur la class pour l'appliquer partout. Je te laisse regarder (et me dire si t'as besoin d'aide). Faire de la doc même basique ça aide à consommer l'api ensuite